### PR TITLE
Enabling ppc64le

### PR DESCRIPTION
**Pull request by Ryan-Dmello**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Ryan-Dmello** [comment]
@conda-forge-admin please rerender

**Comment by Ryan-Dmello** [comment]
@conda-forge-admin, please ping team

**Comment by Ryan-Dmello** [comment]
@conda-forge-admin, please ping team

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ source:
   sha256: 66f454ada0f56eae43066f604a222b09893f98c1adc18df169710761b8f32fe8 
 
 build:
- number: 0
+ number: 2
```
Comment (isuruf) [inline recipe/meta.yaml]: 1?

Comment (Ryan-Dmello) [reply recipe/meta.yaml]: updated

**Comment by Ryan-Dmello** [comment]
@conda-forge-admin please rerender

**Comment by Ryan-Dmello** [comment]
@conda-forge-admin please rerender

**Comment by Ryan-Dmello** [comment]
@conda-forge-admin please rerender

**Comment by Ryan-Dmello** [comment]
@conda-forge-admin please rerender

**Comment by Ryan-Dmello** [comment]
Hi @isuruf, can you please let me know if any more changes are needed for this PR to get merged?

Thanks,
Ryan D'Mello

File: conda-forge.yml [hunk 2]
```diff
@@ -6,4 +6,6 @@ appveyor:
     BINSTAR_TOKEN: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1
 provider:
   win: azure
+  linux_ppc64le: azure
+  linux_aarch64: azure
```
Comment (isuruf) [inline conda-forge.yml]: Can you use `default` instead of `azure`?

Comment (Ryan-Dmello) [reply conda-forge.yml]: setting to default triggers a build to travis-ci.org, which errors out without initiating a build. Hence had modified to use azure. Changed back to "default" as per suggestion

Comment (Ryan-Dmello) [reply conda-forge.yml]: @isuruf, I've updated the value to default as suggested

**Pull request merged**